Cargo.toml

```diff
@@ -67,7 +67,7 @@ ibc-core            = { version = "0.51.0", path = "./ibc-core", default-feature
 ibc-clients         = { version = "0.51.0", path = "./ibc-clients", default-features = false }
 ibc-apps            = { version = "0.51.0", path = "./ibc-apps", default-features = false }
 ibc-primitives      = { version = "0.51.0", path = "./ibc-primitives", default-features = false }
-ibc-derive          = { version = "0.6.0", path = "./ibc-derive" }
+ibc-derive          = { version = "0.6.1", path = "./ibc-derive" }
```

Comment:
Do we add in our changelog that ibc-derive is updated?

Reply:
Haven't been doing so, but good to include.

Reply:
f827010